packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -204,6 +205,29 @@ async function searchImages(value: string): Promise<string[]> {
   });
   return result;
 }
+
+let latestTagMessage: string | undefined = undefined;
+async function searchLatestTag(): Promise<void> {
+  if (imageNameIsInvalid || !imageToPull || imageToPull.includes(':')) {
+    latestTagMessage = undefined;
+    return;
+  }
+  try {
+    let image = imageToPull;
+    if (image.startsWith(DOCKER_PREFIX + '/')) {
+      image = image.slice(DOCKER_PREFIX.length + 1);
+    }
+    const tags = await window.listImageTagsInRegistry({ image });
+    const latestFound = tags.includes('latest');
+    if (!latestFound) {
+      latestTagMessage = '"latest" tag not found. You can search a tag by typing ":"';
```

Comment:
question: wondering about typing/appending/adding

Reply:
Appending seems more accurate to me. I'll change